readme.md

```diff
@@ -31,6 +31,7 @@ use \Mission4\CinnamonRole\Traits\Rolable;
 ```
 And register your policies in the **AuthServiceProvider** `boot()` method.
 ``` php
+use \Mission4\CinnamonRole\CinnamonRole;
```

Comment:
Just add a reminder at the bottom of this code block to remember to import the class.